test/dlc_tests/container_tests/bin/benchmark/run_mxnet_inference_performance_cpu

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+TIMESTAMP=$(date "+%Y-%m-%d-%H-%M-%S")
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs/cpu
+LOG_FILE=inference_results_${COMMIT_INFO}_${TIMESTAMP}.txt
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+set -e
+pip install awscli
+echo Fetch s3 bucket >&2
+aws s3 sync s3://aws-ml-platform-datasets/imagenet/pass-through/ /home/ubuntu/data
+echo Fetch finished >&2
+git clone https://github.com/awslabs/deeplearning-benchmark.git ${HOME_DIR}/artifacts/mxnet/deeplearning-benchmark  && cd ${HOME_DIR}/artifacts/mxnet/
+START=$(date +%s)
+pip install psutil gluoncv && export KMP_AFFINITY='granularity=fine,compact,1,0' && export OMP_NUM_THREADS=36 && python deeplearning-benchmark/image_classification/infer_imagenet.py \
+--use-rec --batch-size 256 --dtype float32 --num-data-workers 40 --mode hybrid --model resnet50_v2 --use-pretrained \
+--kvstore local --log-interval 1 --rec-val /home/ubuntu/data/val-passthrough.rec --rec-val-idx /home/ubuntu/data/val-passthrough.idx 2>&1 | tee ${LOG_DIR}/"${LOG_FILE}"
+python /test/bin/benchmark/get_mxnet_avg_throughput.py ${LOG_DIR}/"${LOG_FILE}" 1 >> ${LOG_DIR}/"${LOG_FILE}"
+END=$(date +%s)
+DIFF=$(( END - START ))
+echo Script took $(( DIFF/60 )) mins to finish >> ${LOG_DIR}/"${LOG_FILE}"
+
+echo Benchmark Results: >&2
+echo MXNet Inference py"${PYTHON_VERSION}" cpu >&2
+tail ${LOG_DIR}/"${LOG_FILE}" >&2 # Dsiplay only the results to console
```

Comment:
Minor spelling mistake in this and some other files.

Reply:
Thanks for pointing it out